subsys/bootloader/bl_client/CMakeLists.txt

```diff
@@ -5,6 +5,3 @@
 #
 zephyr_library()
 zephyr_library_sources(bl_client.c)
-zephyr_link_libraries(nrfxlib_crypto)
```

Comment:
This change is not explained in the commit message. Why are these removed?

Reply:
They were needed earlier for #include files, but that has been fixed/changed a while ago, but these weren't removed then. I'll see if I can mention it in the commit message.

Reply:
Added some explanation in the commit message.